build.gradle

```diff
@@ -29,10 +29,6 @@ repositories {
     }
 }
 
-kotlin {
```

Comment:
We should add
```
    th2JavaRelease {
        targetJavaVersion.set(JavaVersion.VERSION_11)
    }
```

Reply:
This JVM target is set by default

Reply:
https://github.com/th2-net/th2-gradle-plugin/blob/master/plugin/src/main/kotlin/com/exactpro/th2/gradle/JavaReleaseTh2Extension.kt#L32

Reply:
I'm not sure than default plugin behavior should decide this question, but OK

Reply:
If we specify the target release in every project we will have to update it later.
The plugin version update can be done automatically on other hand and does not require going through all the repositories one by one